pandas/tests/tools/test_to_datetime.py

```diff
@@ -1225,15 +1228,18 @@ def test_iso_8601_strings_with_different_offsets_utc(self):
 
     def test_iso8601_strings_mixed_offsets_with_naive(self):
         # GH 24992
-        result = to_datetime(
+        # Can't parse consistently, need to parse each element in loop.
```

Comment:
Similar comment as before - unless there is some value in these items being inconsistent that is specific to this test I think better to just cleanse the inputs

Reply:
thanks @WillAyd for your review! I've removed these two tests, and cleaned the inputs for another one where the purpose of the test isn't related to having mixed inputs